ibm_db2/README.md

```diff
@@ -47,7 +47,7 @@ For Agent versions >= 7.0 and < 7.58:
 For Agent versions >= 7.58:
 
 ```text
-"C:\Program Files\Datadog\Datadog Agent\embedded3\python.exe" -m pip install ibm_db==3.2.2
+"C:\Program Files\Datadog\Datadog Agent\embedded3\python.exe" -m pip install ibm_db==3.2.3
```

Comment:
Do we need this bump for the error to go away? If not, do you mind making a separate PR for that?

Reply:
Yes, it is needed otherwise the pip installation will fail stating that it would require "Microsoft C++ Build Tools". More context in AGENT-12576.

As per testing, on DD Agent 7.58.x / Python 3.12 / Windows 2016, I get the following results:

- pip install of  ibm_db == 3.1.4 / 3.2.0 / 3.2.1 / 3.2.2

  ```
        error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

    note: This error originates from a subprocess, and is likely not a problem with pip.
    ERROR: Failed building wheel for ibm_db
  Failed to build ibm_db
  ERROR: Could not build wheels for ibm_db, which is required to install pyproject.toml-based projects
  ```
   Note that I did get it installed after installing "Microsoft C++ Build Tools" but it was a bit of a hassle getting .NET Framework updated to 4.8 since the BuildTools installer wouldn't launch with .NET 4.6.

- pip install of ibm_db == 3.2.3 - installs without "Microsoft C++ Build Tools" but gets `unable to import module`

Reply:
Done. README update addressed in PR https://github.com/DataDog/integrations-core/pull/18944